canonical-websites/www.ubuntu.com
# 3839: Build German translated VMWare to OpenStack page

## Done

- Built a German translated VMWare to OpenStack page at /engage/de/vmware-to-openstack
- Added a German translated version of the Contact Us form page
- Added navigation links between English and German versions

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8001/engage/de/vmware-to-openstack
- Run through the following [QA steps](https://github.com/canonical-webteam/practices/blob/master/workflow/qa-steps.md)
- Check that the page matches the [copydoc](https://docs.google.com/spreadsheets/d/1R5PY7Sr7u8aJxX4QzjXHwnqcziPh5MZuGLUfjSD-6qg/edit#gid=0)
- Check that the navigation links link the English and German versions
- Click the "Kontaktieren Sie einen Cloud-Experten!" button near the bottom of the page
- Check that it takes you to a German translated version of the /cloud/contact-us form
- Check that the form submits

## Issue / Card

Fixes #3839, fixes #3840
